DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -73,11 +73,11 @@
    "source": [
```

Comment:
<p>Instead of "Only 2 shape definitions", should we say "Only two shape definitions"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>